xtheadmaee.adoc

```diff
@@ -0,0 +1,34 @@
+[#xtheadmaee]
+== Extend MMU address attribute
+
+[NOTE,caption=Frozen]
+The `XTheadMaee` extension is `stable`.
```

Comment:
We should probably clarify that XTheadMaee is a `non-standard non-conforming extension`.
The PTE bits that are used by MAEE are reserved for standard usage.
As far as I understood, this was caused by the fact that RVI did not have a standard yet and T-Head needed a solution.

Reply:
Yes, when c906 came to the world, RVI didn't have the standard with similar function. But now RVI has ratify the SVPBMT, which has the similar function with XTheadMaee. c908 supports both XTheadMaee and SVPBMT. And c907 and other newer CPUs only support SVPBMT.

Reply:
> We should probably clarify that XTheadMaee is a `non-standard non-conforming extension`. The PTE bits that are used by MAEE are reserved for standard usage. As far as I understood, this was caused by the fact that RVI did not have a standard yet and T-Head needed a solution.
Yes, when c906 came to the world, RVI didn't have the standard with similar function. But now RVI has ratify the SVPBMT, which has the similar function with XTheadMaee. c908 supports both XTheadMaee and SVPBMT. And c907 and other newer CPUs only support SVPBMT.

Reply:
> We should probably clarify that XTheadMaee is a `non-standard non-conforming extension`. The PTE bits that are used by MAEE are reserved for standard usage. As far as I understood, this was caused by the fact that RVI did not have a standard yet and T-Head needed a solution.
Yes, when c906 came to the world, RVI didn't have the standard with similar function. But now RVI has ratify the SVPBMT, which has the similar function with XTheadMaee. c908 supports both XTheadMaee and SVPBMT. And c907 and other newer CPUs only support SVPBMT